pyproject.toml

```diff
@@ -315,7 +315,6 @@ module = [
     "sentry.api.serializers.rest_framework.doc_integration",
     "sentry.api.serializers.rest_framework.mentions",
     "sentry.api.serializers.rest_framework.notification_action",
-    "sentry.api.serializers.rest_framework.project",
```

Comment:
did you mean to delete this?

Reply:
oh whoops no

Reply:
actually I think it was a bot, should be fine tho to use strict typing

Reply:
oh interesting okay

Reply:
yep this is great!  it means you fixed the remaining errors in that particular file so now it is being checked normally :tada: